README.md

```diff
@@ -187,8 +187,6 @@ To embed the Sign-in Widget via CDN, include links to the JS and CSS files in yo
 
 The CDN URLs contain a version number. This number should be the same for both the Javascript and the CSS file and match a version on the [releases page](/releases).
 
-The standard JS asset served from our CDN includes polyfills via [`@babel/polyfill`](https://babeljs.io/docs/en/babel-polyfill/) to ensure compatibility with older browsers. This may cause conflicts if your app already includes polyfills. For this case, we provide an alternate JS asset which does not include any polyfills.
```

Comment:
Instead of removing this paragraph, we probably can mention something similar to 
> As of Babel 7.4.0, this package has been deprecated in favor of directly including core-js/stable (to polyfill ECMAScript features) and regenerator-runtime/runtime (needed to use transpiled generator functions):

Quote from https://babeljs.io/docs/en/usage#polyfill